### PR TITLE
[web][SIM-167] Select Program Institute

### PR DESCRIPTION
**Pull request by fwpushan**
1. Create basic layout of select-program page
2. Create Institute List view
3. Create Institute service
4. Fix lint issues

Screenshot 2021-02-19 at 3 48 40 AM

### CODE REVIEW

File: sources/packages/web/src/components/fa-application/InstituteList.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        @change="onChange"
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { SetupContext, onMounted, ref } from "vue";
+import { InstituteService, Institute } from "../../services/InstituteService";
+// import Dropdown from "primevue/dropdown";
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/fa-application/InstituteList.vue:19]: Please remove the commented line.

File: sources/packages/web/src/components/fa-application/InstituteList.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+export default {
+  emits: ["update:modelValue", "change"],
+  /*components: {
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/fa-application/InstituteList.vue:23]: Please remove the commented line.

**Review by andrewsignori-aot — APPROVED** [review]
Please remove the commented code. Besides that looks good to me 👍

File: sources/packages/web/src/components/fa-application/InstituteList.vue (hunk 1 above)
Comment (fwpushan) [reply sources/packages/web/src/components/fa-application/InstituteList.vue]: DONE

**Comment by fwpushan** [comment]
> Please remove the commented code. Besides that looks good to me 👍

DONE

**Review by 2011sandeepraj — APPROVED** [review]
LGTM

**Pull request merged**